gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/TaskUpdateNotifier.java

```diff
@@ -6,34 +6,38 @@
  */
 package com.farao_community.farao.gridcapa.task_manager.app;
 
+import com.farao_community.farao.gridcapa.task_manager.api.TaskDto;
 import com.farao_community.farao.gridcapa.task_manager.app.entities.Task;
-import org.springframework.cloud.stream.function.StreamBridge;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.messaging.simp.SimpMessagingTemplate;
 import org.springframework.stereotype.Service;
 
+import java.time.format.DateTimeFormatter;
 import java.util.Set;
 
 /**
  * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
  */
 @Service
 public class TaskUpdateNotifier {
-    private static final String TASK_UPDATED_BINDING = "task-updated";
-    private static final String TASK_STATUS_UPDATED_BINDING = "task-status-updated";
```

Comment:
The notification on this binding is important for other micro-services than front-end app, so it should not be removed

Reply:
I completely remove the notification with rabbitMQ. If I understand you we have to keep the rabbitMQ and the stomp notification. it's mean no improvement for the rabbitMQ server, we will still flood queues in case of importing annual files

Reply:
Yes and no. First, the task-status-updated notifies only when the status of the task changes, so in theory it notifies less than task-updated, but in real life when we import yearly file it might change the 8760 task status so we will still have these 8760 notifications. But, the real bottleneck was the task-notification-server, to send messages on websocket so you will still fix this issue with your development. With these task status updates it will remain only rabbitmq messages read by back-end services and this is not slow operations. So, all good.

Reply:
notification with the streamBridge has return